2021-09-08 13:25:42,157 INFO    MainThread:31036 [wandb_setup.py:_flush():69] setting env: {}
2021-09-08 13:25:42,158 INFO    MainThread:31036 [wandb_init.py:_log_setup():342] Logging user logs to C:\Programming\rl_cooperation\Pygame_RL_cooperation\Algorithm\DQN_SpaceCannons_PER\wandb\offline-run-20210908_132542-1wb35vun\logs\debug.log
2021-09-08 13:25:42,159 INFO    MainThread:31036 [wandb_init.py:_log_setup():343] Logging internal logs to C:\Programming\rl_cooperation\Pygame_RL_cooperation\Algorithm\DQN_SpaceCannons_PER\wandb\offline-run-20210908_132542-1wb35vun\logs\debug-internal.log
2021-09-08 13:25:42,160 INFO    MainThread:31036 [wandb_init.py:init():375] calling init triggers
2021-09-08 13:25:42,160 INFO    MainThread:31036 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-08 13:25:42,161 INFO    MainThread:31036 [wandb_init.py:init():424] starting backend
2021-09-08 13:25:42,161 INFO    MainThread:31036 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-09-08 13:25:42,183 INFO    MainThread:31036 [backend.py:ensure_launched():135] starting backend process...
2021-09-08 13:25:42,697 INFO    MainThread:31036 [backend.py:ensure_launched():139] started backend process with pid: 29320
2021-09-08 13:25:42,700 INFO    MainThread:31036 [wandb_init.py:init():429] backend started and connected
2021-09-08 13:25:42,705 INFO    MainThread:31036 [wandb_init.py:init():477] updated telemetry
2021-09-08 13:25:42,709 INFO    MainThread:31036 [wandb_init.py:init():540] starting run threads in backend
2021-09-08 13:25:43,910 INFO    MainThread:31036 [wandb_run.py:_console_start():1601] atexit reg
2021-09-08 13:25:43,912 INFO    MainThread:31036 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-08 13:25:43,912 INFO    MainThread:31036 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-08 13:25:43,916 INFO    MainThread:31036 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-08 13:25:43,917 INFO    MainThread:31036 [wandb_init.py:init():565] run started, returning control to user process
2021-09-08 13:25:59,975 INFO    MainThread:31036 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-09-08 13:25:59,975 INFO    MainThread:31036 [wandb_run.py:_restore():1543] restore
2021-09-08 13:26:00,239 INFO    MainThread:31036 [wandb_run.py:_wait_for_finish():1693] got exit ret: 
2021-09-08 13:26:00,348 INFO    MainThread:31036 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}

